### DISCUSSION TITLE
Still waiting for first payout

### DISCUSSION THREAD

**Discussion by torshid** [community/community#56114] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

It's been more than 3 months since my first sponsorship, but nothing was sent to my payout account so far.

Is there something missing on my side? Or is there a problem with GitHub/Stripe?

Thanks.

**Comment by DHayes10** [comment, marked as answer]
Hi @torshid, 

Thank you for reaching out. I have issued a manual payout which is estimated to arrive in the verified Stripe account listed in your profile on May 24th. 

Best, 
GitHub Sponsors

**Reply by torshid** [reply]
Thank you!

**Comment by kiliczsh** [comment]
Hi @DHayes10, I have same problem waiting for first payout for 1 month. It was displaying May 22 but now updated to June 22. I would appreciate your help.

**Reply by torshid** [reply]
You need to wait 60 days after the first payment to receive your first payout https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing.

**Reply by kiliczsh** [reply]
thank you for information @torshid

**Reply by kiliczsh** [reply]
Nope still not @lsgdefault

**Comment by maekawatoshiki** [comment]
Hi, it seems like I have the same problem. It has been over two or three months since the first sponsorship.
Any help would be appreciated.

**Reply by DeveloppeurPascal** [reply]
The same for me : a payment in January, visible on Stripe but in "funds pending transfer via GitHub" state. Do you know if they have a minimum amount for payments ? I haven't seen anything about that in the doc.

**Reply by Rinshin-Jalal** [reply]
there is a limit of 100$ minimum
